src/lib/controllers/app-controller.js

```diff
@@ -1,16 +1,24 @@
 import App from '@/vue/app.vue'
 import Vue from '@vue-runtime'
+
 import TextsController from '@/lib/controllers/texts-controller.js'
 
+import L10n from '@/lib/l10n/l10n.js'

```

Comment:
hmm. can't we import it from alpheios-core?

Reply:
We would need to import the whole alpheios-core with its all dependencies - and it is really a huge amount of files.
I think that this module should be a separate package that could be used in any repo without importing alpheios-core.

But while it is inside alpheios-core I simply copied these three files from l10n package.
do you think that it is worth to import the whole alpheios-core with all its dependencies?

Reply:
I guess we can wait. We should probably make l10n a distributable package so that we can do this though. I'll enter an issue for that and it can stay as you have it until then. thanks.